src/Featurizers/Components/FrequencyEstimator.h

```diff
@@ -108,6 +107,8 @@ template <
 >
 using FrequencyEstimator                       = TrainingOnlyEstimatorImpl<Details::FrequencyTrainingOnlyPolicy, MaxNumTrainingItemsV>;
 
+// Normally we want the featurizer name at the top of the file, but GrainFeaturizers are just type defs, so we want the name next to it.
+static constexpr char const * const         GrainFrequencyEstimatorName("GrainFrequencyEstimator");

```

Comment:
I moved this down, but would I be better to move the name and the typedef up instead? Just a thought.

Reply:
This looks good to me.

---
In reply to: [391819433](https://github.com/microsoft/FeaturizersLibrary/pull/165#discussion_r391819433) [](ancestors = 391819433)